eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -206,6 +206,9 @@ public function lookup($url)
                 $this->logger->logCall(__METHOD__, array('url' => $url));
                 $urlAlias = $this->persistenceHandler->urlAliasHandler()->lookup($url);
                 $cache->set($urlAlias->id);
+
+                $urlAliasCache = $this->cache->getItem('urlAlias', $urlAlias->id);
```

Comment:
Hmm on a brief look I would expect the 'urlAlias', 'url' to (also) be set here, but could be wrong...